C/manual/tips-appendix.docbook

```diff
@@ -246,122 +232,119 @@
 
             <row>
               <entry>
-                <para>BAMOSZ funds, HU
+                <para>Bloomberg
```

Comment:
Sorry, I forgot to replay earlier.

 I intentionally left the region out here and in a few other places (`marketwatch`, `stockdata`, `twelvedata`). This was to follow what has been done in `financeapi` and, to some degree, `goldmoney`. I guess adding some extra term here should be followed in other places, right?

As for using term `multi`: I'd refrain from doing that, as there are already "Multiple Quote Sources" and this may cause confusion. `global` perhaps?

Reply:
The TLD isn't useful, there are too many supposedly country TLDs that have been commercialized. `supranational` seems a bit legalistic and `global` implies a bit wider scope than is the case. How about `multnational`, a term widely used in the US for companies doing business in more than one country?